emperor/support_files/js/view.js

```diff
@@ -694,6 +856,19 @@ DecompositionView.prototype.setColor = function(color, group) {
     });
     cloud.geometry.attributes.color.needsUpdate = true;
   }
+  else if (UIState['view.viewType'] == 'parallel-plot' &&
+           this.decomp.isScatterType()) {
+    var lines = this.markers[0];
+    color = new THREE.Color(color);
+    var numPoints = (this.decomp.dimensions * 2 - 2);
+    group.forEach(function(plottable) {
+    var startIndex = plottable.idx * numPoints;
```

Comment:
Would you mind indenting the code in this function?

Reply:
Done